src/Simulation/EntryPointDriver/Azure/AzureSettings.cs

```diff
@@ -0,0 +1,183 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using Microsoft.Azure.Quantum;
+using Microsoft.Azure.Quantum.Authentication;
+using Microsoft.Quantum.Runtime.Submitters;
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Quantum.EntryPointDriver.Azure
+{
+    using Environment = System.Environment;
+
+    /// <summary>
+    /// Settings for a submission to Azure Quantum.
+    /// </summary>
+    public sealed class AzureSettings
```

Comment:
Makes sense.

Reply:
This is interesting. In general I also like to avoid redundancy and like concise naming. But always keep in mind that the most important tool to study codebase is grep (text search). If someone who's not familiar with codebase will search for 'settings', it would be a disaster as this is a very common word. AzureSettings is concise enough and rather unique at the same time so search will work well.

Reply:
I am spoilt by often using FindAllReferences opposed to grep... Either one is fine by me - I'll leave it up to you.

Reply:
Looking more closely into this, this was an already existing class. The only thing that happened is that its implementation was moved into its own file. When moving the implementation into its own file, the namespace changed from `Microsoft.Quantum.EntryPointDriver` to `Microsoft.Quantum.EntryPointDriver.Azure`. Given that these were existing public classes, I'll revert to the original namespace and keep the name as is.

Reply:
Fixed the namespace backwards compatibility problem.